server/data_stream.py

```diff
@@ -59,7 +59,7 @@ def _get_ops_by_type(commit: models.ComAtprotoSyncSubscribeRepos.Commit) -> dict
 
 
 def run(name, operations_callback, stream_stop_event=None):
-    while True:
+    while stream_stop_event and not stream_stop_event.is_set():
```

Comment:
idk why but `stream_stop_event` could be `None`. so, in this case, we expect endless loop

check `while stream_stop_event` doesn't allow support so

maybe we need delete default `None` for `stream_stop_event`